.github/workflows/base.yml

```diff
@@ -0,0 +1,45 @@
+name: base
+
+on:
+  schedule:
+    - cron: '0 15 * * *'
+  pull_request:
+    paths:
+      - './**'
```

Comment:
I think this also matches directories in the root but maybe that's okay (not triggered by changes in files within those directories)?